Src/Shaders/PBREffect.fx

```diff
@@ -0,0 +1,294 @@
+// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
+// PARTICULAR PURPOSE.
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// http://go.microsoft.com/fwlink/?LinkID=615561
+
+
+Texture2D<float4> AlbedoTexture : register(t0);
+Texture2D<float3> NormalTexture : register(t1);
+Texture2D<float3> RMATexture    : register(t2);
+
+Texture2D<float3> EmissiveTexture : register(t3);
+
+TextureCube<float3> RadianceTexture   : register(t4);
+TextureCube<float3> IrradianceTexture : register(t5);
+
+sampler SurfaceSampler : register(s0);
+sampler IBLSampler     : register(s1);
+
+cbuffer Constants : register(b0)
+{
+    float3   EyePosition            : packoffset(c0);
+    float4x4 World                  : packoffset(c1);
+    float3x3 WorldInverseTranspose  : packoffset(c5);
+    float4x4 WorldViewProj          : packoffset(c8);
+    float4x4 PrevWorldViewProj      : packoffset(c12);
+
+    float3 LightDirection[3]        : packoffset(c16);
+    float3 LightColor[3]            : packoffset(c19);   // "Specular and diffuse light" in PBR
+ 
+    float3 ConstantAlbedo           : packoffset(c22);   // Constant values if not a textured effect
+    float  Alpha                    : packoffset(c22.w);
```

Comment:
Added alpha value that can be used in 'constant albedo' scenarios as well as a scale for the alpha channel in albedo texture